src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ServiceUserDTO.kt

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceUserData
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Sex
+import java.util.Date
+
+data class ServiceUserDTO(
+  var pncNumber: String? = null,
+  var crn: String? = null,
+  var nomisNumber: String? = null,
+  var dob: Date? = null,
+  var sex: Sex? = null,
+  var ethnicity: String? = null,
+  var title: String? = null,
+  var firstName: String? = null,
+  var lastName: String? = null,
+  var otherNames: String? = null,
+  var address: String? = null,
+  var needs: String? = null,
+  var preferredLanguage: String? = null,
+  var religionOrBelief: String? = null,
+  var sexualOrientation: String? = null,
+  var disabilities: String? = null,
```

Comment:
I'm worried about creating this detailed API spec now. I think for the skeleton we may go with CRN, name and responsible LDU.

From a data handling perspective, we shouldn't copy more data than we need for processing the referrals.

Could we skip fields until we have a need for them through the UI or some business logic?
Is there added complexity if we want to add fields later?

Reply:
i just think we're making more work for ourselves down the line. i'm basing this on what's in the prototype, and all these fields are included. if something has changed we can remove the ones that aren't needed.